src/frontend/Sider/Behandling/Venstremeny/Oppsummering/BarnDetaljer.tsx

```diff
@@ -9,17 +9,43 @@ import {
     typeBarnepassTilTekst,
     årsakBarnepassTilTekst,
 } from '../../../../typer/behandling/behandlingFakta/faktaBarn';
-import { JaNei } from '../../../../typer/common';
+import { JaNei, jaNeiTilTekst } from '../../../../typer/common';
 import { tekstEllerKode } from '../../../../utils/tekstformatering';
 
 const BarnDetaljer: React.FC<{ barn: FaktaBarn }> = ({ barn }) => {
     const typePass = barn.søknadgrunnlag?.type;
     const startetIFemte = barn.søknadgrunnlag?.startetIFemte;
+    const utgifter = barn.søknadgrunnlag?.utgifter;
     const årsak = barn.søknadgrunnlag?.årsak;
 
     return (
         <InfoSeksjon label={`${barn.registergrunnlag.navn} ${barn.ident}`} ikon={<ChildEyesIcon />}>
             <BodyShort size="small">{tekstEllerKode(typeBarnepassTilTekst, typePass)}</BodyShort>
+            {utgifter !== undefined && (
+                <>
+                    {utgifter.harUtgifterTilPass === JaNei.JA ? (
```

Comment:
Tror dette burde renames til 
```suggestion
                    {utgifter.harUtgifterTilPassHelePerioden === JaNei.JA ? (
```